source/extensions/http/stateful_session/cookie/cookie.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package envoy;
+
+message Cookie {
+  string address = 1;
+  uint32 expires = 2;
```

Comment:
uint64 would better for timestamp, even the unit is second.

Reply:
Good idea. Thanks. Will change the code.